### ISSUE TITLE
Modal contact picture menu is barely visible

### ISSUE THREAD

**Issue by jkischel** [nextcloud/contacts#1086]
### Steps to reproduce
1. create a contact
2. upload a picture

### Expected behaviour
It is somehow possible to edit or delete the contact picture

### Actual behaviour
Clicking the image just shows the actual picture, but without any change options.

### Server configuration
**Nextcloud version:** (see Nextcloud admin page)
15.0.7

**Contacts version:** (see Nextcloud apps page)
3.1.1

**Updated from an older Nextcloud or fresh install:**
fresh install

```
i think, all this login and log stuff is not necessary here. I've seen the feature of editing contact pictures described in a manual I was googling for, but the buttons in the picture view simply aren't present. Not visible, and also nothing that might be suspected to do the job in the HTML source.
```

### Client configuration
**Browser:**
Edge, Firefox, Vivaldi

**Comment by skjnldsv** [comment]
Hello!
Have you tried clicking the picture?
It will open the picture and you will be able to edit it :)
![Capture d’écran_2019-05-02_16-14-17](https://user-images.githubusercontent.com/14975046/57081597-5b88f380-6cf5-11e9-9912-b54d53cd9b46.png)

**Comment by jkischel** [comment]
woah, okay. Finally found them.

The three points icon is on all my devices (HP laptop, Android Smartphone, Home PC) very well hidden as it has nearly the same color as the background. I'll fix this locally by adding some custom CSS, but this is still a usability issue.

In the manual, they were listed as icons at the bottom, but of course not that critical.

**Comment by skjnldsv** [comment]
Ah yes, I can indeed reproduce the issue.
I'll take a look. I guess this needs a fix!

Thanks for reporting :)

**Comment by OverByThere** [comment]
Can also confirm this is happening on brand new install and confused me for a few mins!

**Comment by skjnldsv** [comment]
Sorry everyone :)
Should be fixed in upcoming version ;)

**Comment by skjnldsv** [comment]
Fixed by #1099
